kvbc/include/merkle_processor/SyncMerkleProcessor.h

```diff
@@ -0,0 +1,43 @@
+// Concord
+//
+// Copyright (c) 2023 VMware, Inc. All Rights Reserved.
+//
+// This product is licensed to you under the Apache 2.0 license (the "License").
+// You may not use this product except in compliance with the Apache 2.0 License.
+//
+// This product may include a number of subcomponents with separate copyright
+// notices and license terms. Your use of these subcomponents is subject to the
+// terms and conditions of the sub-component's license, as noted in the
+// LICENSE file.
+
+#pragma once
+
+#include "BaseMerkleProcessor.h"
+
+namespace concord {
+namespace kvbc {
+namespace sparse_merkle {
+
+using concordUtils::bufferToHex;
+
+class SyncMerkleProcessor : public BaseMerkleProcessor {
+ public:
+  virtual ~SyncMerkleProcessor() = default;
+
+  virtual void Init(uint numVersionsStored) override;
+
+  virtual void BeginVersionUpdateBatch() override;
+  virtual void UpdateAccountTree(const address& addr, const std::string& key, const value& data) override;
+
+  virtual void CommitVersionUpdateBatch() override;
+
+  virtual std::vector<std::string> GetAccountMerkleRootPath(const address& addr) override;
```

Comment:
what does root path mean? full path from root?

Reply:
This call returns the Merkle path in the Global State MPT from the GS Merkle root to the address's tree Merkle root. The next one - GetAccountStorageKeyMerklePath returns the path from the key, the value of which we want to verify, to the address's tree Merkle root.